pkg/genericapiserver/genericapiserver.go

```diff
@@ -475,6 +477,14 @@ func (s *GenericAPIServer) init(c *Config) {
 
 	attributeGetter := apiserver.NewRequestAttributeGetter(s.RequestContextMapper, s.NewRequestInfoResolver())
 	handler = apiserver.WithAuthorizationCheck(handler, attributeGetter, s.authorizer)
+	if len(c.AuditLog) != 0 {
+		auditLog, err := os.OpenFile(c.AuditLog, os.O_APPEND|os.O_WRONLY, 0644)
```

Comment:
I believe `O_CREATE` also should be allowed, in cases where the file doesn't exists already? When running with Hyperkube I get the following error
> F0707 12:42:48.410710       1 genericapiserver.go:483] Could not open audit log file: open /kube-audit.log: no such file or directory